clarify unmentioned constraint port.name equalness (cherry-pick #11680) (#11891)
istio/istio.io: content/en/docs/reference/glossary/namespace-sameness.md

Within a multicluster mesh, [namespace sameness](https://github.com/kubernetes/community/blob/master/sig-multicluster/namespace-sameness-position-statement.md)
applies and all namespaces with a given name are considered to be the same namespace. If multiple clusters contain a
`Service` with the same namespaced name, they will be recognized as a single combined service. By default, traffic is
load-balanced across all clusters in the mesh for a given service.

> For one or many ports within the combined service, same port value should also have same port name.

Reviewer: ```suggestion
Ports that match on _number_ must also have the same port _name_ to be considered as a combined service port.```

Author: Thanks for reply. I updated, please have a look.
